templates-java/src/main/java/br/com/templates_java/config/jms_aws_sqs/ProducerMessageGenerator.java

```diff
@@ -0,0 +1,24 @@
+package br.com.templates_java.config.jms_aws_sqs;
+
+
+import br.com.generator.core.Generator;
+import br.com.generator.core.GeneratorOptions;
+
+import java.io.File;
+import java.io.IOException;
+
+public class ProducerMessageGenerator extends Generator {
```

Comment:
The class `ProducerMessageGenerator` should probably be `public final` since it's not designed to be extended, or provide a reason why it's not final.